packages/web3-types/src/eth_types.ts

```diff
@@ -45,8 +45,11 @@ export enum BlockTags {
 	EARLIEST = 'earliest',
 	LATEST = 'latest',
 	PENDING = 'pending',
+	SAFE = `safe`,
+	FINALIZED = `finalized`,
```

Comment:
```suggestion
	SAFE = 'safe',
	FINALIZED = 'finalized',
```
should stick to one format